### ISSUE TITLE
Reveal in Explorer shortcut keyboard doesn't work

### ISSUE THREAD

**Issue by fesouzamsft** [microsoft/vscode#100787]
Issue Type: <b>Bug</b>

1 - Select a tab/file among the opened files
2 - Press Shift-Alt-R

Expected: File explorer window opened at the file's folder
Actual: Nothing happens

Note: Using "reveal in explorer" with the mouse works

VS Code version: Code 1.42.1 (c47d83b293181d9be64f27ff093689e8e7aed054, 2020-02-11T14:45:59.656Z)
OS version: Windows_NT x64 10.0.18363

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz (12 x 3696)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.84GB (14.52GB free)|
|Process Argv|.|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (10)</summary>

Extension|Author (truncated)|Version
---|---|---
gitlens|eam|10.2.2
prettier-vscode|esb|5.1.0
vscode-peacock|joh|3.7.2
chat|kar|0.35.0
graphql-for-vscode|kum|1.15.3
vsliveshare|ms-|1.0.2354
vsliveshare-audio|ms-|0.1.85
vsliveshare-pack|ms-|0.4.0
teams-dyson|tea|0.0.2020021302
alphabetical-sorter|ue|2.0.1

</details>

**Comment by ArturoDent** [comment]
Note that that command and keybinding have this context:

`"when": "!editorFocus"`  // so won't work when editor is focussed.

You could delete or modify that `when` clause to get it to work where you want. 

 It is odd that in essence the context menu option has a less restrictive context than the keybinding it offers.

**Comment by gjsjohnmurray** [comment]
Previously discussed in #96893, which is now locked.

I just noticed that `Alt+R` when focused on the editor toggles the `Use Regular Expression` state of the Find (Ctrl+F) widget. If that widget isn't currently displaying the Alt+R still acts on it, making the three toggle buttons briefly visible. I find this a bit weird, but no doubt it was done for a reason. `Alt+C` and `Alt+W` toggle the other two buttons on the widget in the same way, but don't clash with menubar entries in the way `Alt+R` does.

Since that widget toggle button grabs `Alt+R`, it's necessary to use `Shift+Alt+R` to drop down the Run menu from the menubar. And without the '!editorFocus` when-clause on the `Shift+Alt+R` keybinding it won't be possible to access the Run menu directly by keyboard when focused on the editor.

Bottom line seems to be, if you don't mind losing direct keyboard access to the Run menu, change the When Expression to blank, as @ArturoDent suggested:

![image](https://user-images.githubusercontent.com/6726799/85340813-b32c8480-b4de-11ea-826e-11ad719c2a16.png)

**Comment by isidorn** [comment]
Thanks for the explanation @gjsjohnmurray 
And due to that we can close this as designed
There are workarounds as suggested above
